tests/rest/client/test_rooms.py

```diff
@@ -1116,6 +1116,320 @@ def test_rooms_messages_sent(self) -> None:
+                "users": {"@sid1:red": 100},
+                "users_default": 0,
+            },
+            self.power_levels(room_id),
+        )
+
+    @unittest.override_config(
+        {
+            "default_power_level_content_override": {
+                "public_chat": {"events": {"custom.event": 0}},
+            }
+        },
+    )
+    def test_power_levels_with_server_override(self) -> None:
+        """
+        With a server configured to modify the room-level defaults,
+        Create a room, without providing any extra power level overrides.
+        Confirm that the room's power levels reflect the server-level overrides.
+
+        Similar to https://github.com/matrix-org/matrix-spec/issues/492,
+        we overwrite each key of power_level_content_override completely.
+        """
+
+        room_id = self.helper.create_room_as(self.user_id)
+        self.assertEqual(
+            {
+                "ban": 50,
+                "events": {
+                    "custom.event": 0,
+                },
+                "events_default": 0,
+                "historical": 100,
+                "invite": 50,
+                "kick": 50,
+                "redact": 50,
+                "state_default": 50,
+                "users": {"@sid1:red": 100},
+                "users_default": 0,
+            },
+            self.power_levels(room_id),
+        )
+
+    @unittest.override_config(
+        {
+            "default_power_level_content_override": {
+                "public_chat": {
+                    "events": {"server.event": 0},
+                    "ban": 13,
+                },
+            }
+        },
+    )
+    def test_power_levels_with_server_and_room_overrides(self) -> None:
+        """
+        With a server configured to modify the room-level defaults,
+        Create a room, providing different overrides.
+        Confirm that the room's power levels reflect both overrides, and
+        choose the room overrides where they clash.
+        """
+
+        room_id = self.helper.create_room_as(
+            self.user_id,
+            extra_content={
+                "power_level_content_override": {"events": {"room.event": 0}}
+            },
+        )
+
+        self.assertEqual(
+            {
+                "ban": 13,  # From server config override
+                "events": {
+                    "room.event": 0,  # From room override
+                },
+                "events_default": 0,
+                "historical": 100,
+                "invite": 50,
+                "kick": 50,
+                "redact": 50,
+                "state_default": 50,
+                "users": {"@sid1:red": 100},
+                "users_default": 0,
+            },
+            self.power_levels(room_id),
+        )
+
+
+class RoomPowerLevelOverridesInPracticeTestCase(RoomBase):
+    """Tests that we can really do various otherwise-prohibited actions
+    based on overriding the power levels in config."""
```

Comment:
Typically in our code style, when a docstring is multiple lines, the second `"""` will be on its own line.

(Alternatively, this used to be one line and `black` made this mistake :P)

Reply:
Fixed in 55e1eccdd